themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2759,18 +2760,27 @@ If omitted, results are sorted according to their search relevance. If there is
 
 If specified more than once, the first parameter is the primary sort order and subsequent parameters control additional sorting criteria.
 
+Special consideration needs to be given if using this in tandem with the **cursor** parameter, see below.
+
 Allowed values: created, custom, delete, id, modified, module, name, package, parent.urn, pending, project, protected, provider.urn, stack, type, urn.
 
 **asc**: Whether to return results in ascending or descending sort order.
 
 **size**: How many results to return at a time.
 
+**paginate**: If requesting multiple pages of results, the pagination method to use. Should be "cursor" or "page"
```

Comment:
```
(Optional) the pagination method for retrieving multiple pages of results. One of `page` (default) for an ordered list of the first 10k results, or `cursor` for iterating through the entire unordered list of results.
```

Reply:
+1 to Evan's comment, but I'd capitalize `The pagination`.

Reply:
[7757cfa](https://github.com/pulumi/pulumi-hugo/pull/3105/commits/7757cfa27983f0980b2ca5f483973854ce3f8f67)